web-client/src/ustc-ui/Icon/CaseIcons.tsx

```diff
@@ -0,0 +1,42 @@
+import { ConsolidatedCaseIcon } from '@web-client/ustc-ui/Icon/ConsolidatedCaseIcon';
+import { Icon } from '@web-client/ustc-ui/Icon/Icon';
+import React from 'react';
+import classNames from 'classnames';
+
+export function CaseIcons({ formattedCase }: { formattedCase: any }) {
+  return (
+    <div
+      className="multi-filing-type-icon"
+      style={{ display: 'flex', flexWrap: 'nowrap', gap: '1rem' }} // USWDS overwrites these styles at certain breakpoints even with !important
+    >
+      <span
+        className={
+          formattedCase.isSealed ? 'visibility-visible' : 'visibility-hidden'
```

Comment:
I keep the icon present but hidden/inaccessible so that spacing is always consistent.